src/components/shared/painting-info-template/image-carousel/image-carousel.jsx

```diff
@@ -48,7 +48,7 @@ export default function ImageCarousel({
                   src={image.url}
                   width={100}
                   height={100}
-                  alt={image.alt}
+                  alt={image.alt || "image carousel"}
```

Comment:
Image carousel doesn't really describe the image. If there isn't a proper way to provide good alt text, you are better off doing `alt=""`. That is the a11y correct way of providing alt text to an image that doesn't necessarily need alt text, like if it is decorative for example.